samples/bluetooth/mesh/silvair_enocean/README.rst

```diff
@@ -0,0 +1,198 @@
+
+EnOcean buttons:
+      Pressed and releasing the button on the EnOcean switch, the button publishes an OnOff message using the configured publication parameters of its model instance, and toggles the LED state on a :ref:`mesh light <bluetooth_mesh_light_lc>` device.
+      Pressing and holding the button on the EnOcean switch publishes a Level message using the configured publication parameters of its model instance, and changes the emitted LED light level on the :ref:`mesh light <bluetooth_mesh_light_lc>` device.
+
+LEDs:
+   During the provisioning process, the LEDs (1 to 4) are used to output the OOB actions.
+   Once the provisioning and configuration are completed, the LEDs are used to reflect the status of actions, and they show the last known OnOff/Level state of the corresponding button.
+   It will not change its emitted LED light level, it will only be on or off.
+
+Configuration
+*************
+
+|config|
+
+Source file setup
+=================
+
+The Silvair EnOcean sample is split into two source files:
+
+* A :file:`main.c` file to handle initialization.
+* One additional file for handling mesh models, :file:`model_handler.c`.
+
+Building and running
+********************
+
+Make sure to enable the Bluetooth mesh in |NCS| before building and testing this sample.
+See :ref:`Bluetooth mesh user guide <ug_bt_mesh>` for more information.
+
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/silvair_enocean`
+
+.. include:: /includes/build_and_run.txt
+
+.. _bluetooth_mesh_silvair_enocean_testing:
+
+Testing
+=======
+
+.. note::
+   The Silvair EnOcean sample cannot demonstrate any functionality on its own, and needs a device with the :ref:`bluetooth_mesh_light_lc` sample running in the same mesh network.
+
+After programming the sample to your development kit, you can test it by using a smartphone with `nRF Mesh mobile app`_ installed.
+Testing consists of provisioning the device and configuring it for communication with the mesh models.
+
+Provisioning the device
+-----------------------
+
+.. |device name| replace:: :guilabel:`Mesh Silvair EnOcean`
+
+.. include:: /includes/mesh_device_provisioning.txt
+
+Configuring the EnOcean switch
+------------------------------
+
+See :ref:`bt_enocean_commissioning` for details on how to configure the EnOcean switch.
+
+Configuring models
+------------------
+
+See :ref:`ug_bt_mesh_model_config_app` for details on how to configure the mesh models with the nRF Mesh mobile app.
+
+Configure the Generic OnOff Client and the Generic Level Client models on each element on the :guilabel:`Mesh Silvair EnOcean` node:
+
+* Bind the model to :guilabel:`Application Key 1`.
+* Set the publication parameters:
+
+  * Destination/publish address: Set the :guilabel:`Publish Address` to the first unicast address of the Mesh Light Fixture node.
+  * Retransmit count: Set the count to zero (:guilabel:`Disabled`), to prevent the model from sending each button press multiple times.
```

Comment:
>  to prevent the model from sending each button press multiple times.

There is no problem with configuring publish retransmits, it will increase robustness and repetitions will be filtered by replay protection.

Reply:
So, no need to mention the Retransmit count in this section at all then? I'll remove it, as what you write here is true in general and should need to be in each sample doc